emhttp/plugins/dynamix/DashStats.page

```diff
@@ -1588,8 +1609,7 @@ let viewportResizeTimeout;
 window.addEventListener('resize', function(){
```

Comment:
viewportResizeTimeout might also be dead code now too. Would be nice to remove a resize event listener.